src/internal/storage/fileset/reader.go

```diff
@@ -74,7 +74,6 @@ func (fr *FileReader) Index() *index.Index {
 
 // Content writes the content of the file.
 func (fr *FileReader) Content(w io.Writer) error {
-	dataRefs := getDataRefs(fr.idx.File.Parts)
-	r := fr.chunks.NewReader(fr.ctx, dataRefs)
+	r := fr.chunks.NewReader(fr.ctx, fr.idx.File.DataRefs)
```

Comment:
This is never modified in the chunk reader right, we just contract the slice without copying anything?

Reply:
We just iterate over the datarefs, and read the fields. No modifications to the data.